service-api/src/main/java/greencity/enums/EventTime.java

```diff
@@ -1,6 +1,6 @@
 package greencity.enums;
 
 public enum EventTime {
-    FUTURE,
```

Comment:
I see that we use `FUTURE` [here](https://github.com/ita-social-projects/GreenCity/blob/dev/core/src/main/resources/templates/core/management_events.html#L176)
Do you think we should update it as well?